livekit-agents/livekit/agents/transcription/tts_forwarder.py

```diff
@@ -230,8 +235,10 @@ def _create_segment(self) -> _SegmentData:
             processed_hyphenes=0,
             ready_future=asyncio.Future(),
         )
+        self._next_segment_index += 1
+        return data
 
-    async def _forward(self, q: asyncio.Queue[rtc.TranscriptionSegment | None]):
+    async def _forward_co(self, q: asyncio.Queue[rtc.TranscriptionSegment | None]):
```

Comment:
`_co` meaning coroutine? where they would be running in parallel?

Reply:
it means coroutine, it is running sequentially. Compared to a task that is spawned using asyncio.create_task.
The synthax I'm doing it here is just to separate both concept